ambry-store/src/main/java/com.github.ambry.store/DiskManager.java

```diff
@@ -325,6 +325,14 @@ boolean addBlobStore(ReplicaId replica) {
         // add new created store into in-memory data structures.
         stores.put(replica.getPartitionId(), store);
         partitionToReplicaMap.put(replica.getPartitionId(), replica);
+        // create a bootstrap-in-progress file to distinguish it from regular stores (the file will be checked during
+        // BOOTSTRAP -> STANDBY transition)
+        File bootstrapFile = new File(replica.getReplicaPath(), BlobStore.BOOTSTRAP_FILE_NAME);
+        if (!bootstrapFile.exists()) {
+          // if not present, create one. (it's possible the bootstrap file exists because node may crash immediately
+          // after the file was created last time)
+          bootstrapFile.createNewFile();
```

Comment:
This file is never deleted.  How do we know when bootstrap completes?

Reply:
Good question. The file is created within Offline-To-Bootstrap transition (if this is a new replica to be added) and will be deleted at the end of Bootstrap-To-Standby transition, which will be implemented in next PR. Replication Manager is supposed to keep checking replication lag of new replica and ensure it has caught up with peer replicas before completing the transition. Once it catches up, we can safely delete the bootstrap file.

Reply:
ok